src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -632,8 +638,100 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
             {
                 await TryHandleTimeoutAsync(invokeTask, CancellationToken.None, throwOnTimeout, timeoutTokenSource.Token, timerInterval, instance, null);
             }
+            
+            // Get the list of filters to invoke if there are any
+            List<IFunctionInvocationFilter> filters = new List<IFunctionInvocationFilter>();
+            FunctionResult functionResult = null;
+            filters = GetFilters(instance, invoker);
+
+            try
+            {
+                await InvokeExecutingFilters(filters, parameterNames, invokeParameters, functionCancellationTokenSource, instance, logger, jobHost, config);
+                await invokeTask;
+                functionResult = new FunctionResult(true);
+            }
+            catch (Exception ex)
+            {
+                functionResult = new FunctionResult(ex);
+                throw;
+            }
+            finally
+            {
+                await InvokeExecutedFilters(filters, parameterNames, invokeParameters, functionCancellationTokenSource, instance, logger, jobHost, config, functionResult);
+            }
+        }
+
+        private static List<IFunctionInvocationFilter> GetFilters(IFunctionInstance instance, IFunctionInvoker invoker)
```

Comment:
Change the parameter types here to GetFilters(MethodInfo, object) and just pass in the instance.FunctionDescriptor.Method and invoker.Instance values.